sdk/ml/azure-ai-ml/azure/ai/ml/_schema/pipeline/component_job.py

```diff
@@ -124,6 +123,8 @@ class CommandSchema(BaseNodeSchema, ParameterizedCommandSchema):
         ],
         required=True,
     )
+    # code is directly linked to component.code, so no need to validate it
+    code = fields.Str(allow_none=True)
```

Comment:
This should be load only?

Reply:
Load only make sense. Here I just follow the original logic. Let me make a change.